frontend/src/features/apartment/ApartmentNewSalePage/LoadedApartmentSalesPage.tsx

```diff
@@ -121,11 +163,37 @@ const LoadedApartmentSalesPage = ({apartment}: {apartment: IApartmentDetails}) =
         if (errors.purchase_price && errors.purchase_price.type === z.ZodIssueCode.custom) {
             setIsWarningModalVisible(true);
             setWarningsGiven((prev) => ({...prev, purchase_price: true}));
+            setWarningMessage("Kauppahinta ylittää laskelman enimmäishinnan.");
```

Comment:
The warning message string here could also be imported from `errorMessages` in `schema.ts`, like most of the other error messages.